packages/model-viewer/src/features/ar.ts

```diff
@@ -446,7 +446,7 @@ configuration or device capabilities');
       model.position.copy(target.position);
       model.updateWorldMatrix(false, true);
 
-      const arraybuffer = await exporter.parse(model);
+      const arraybuffer = await exporter.parseAsync(model);
```

Comment:
Actually, I believe there's another `exporter.parse` in `scene-graph.ts` as well - could you update that as well for consistency?

Reply:
Yes, I'll take a look.

Reply:
@elalish I don't believe the call in `scene-graph.ts` needs to be altered as it's using the synchronous callback-based version of `parse`. `parse` previously used to support both a callback and Promise-based overload, but now the asynchronous version is called `parseAsync`. `ar.ts` await-ed the return, so it needed to be adjusted to use the new async version, but the call to `parse` in `scene-graph.ts` is callback-based.

Reply:
okay, thanks.